docs/custom-extensions.md

```diff
@@ -0,0 +1,112 @@
+# Add custom PostgreSQL extensions
+
+One of the specificPostgreSQL features is the ability to provide it with additional functionality via [Extensions](https://www.postgresql.org/download/products/6-postgresql-extensions/). Percona Distribution for PostgreSQL [supports a number of extensions](https://docs.percona.com/postgresql/16/), making this list available for the database cluster managed by the Operator as well.
+
+Still there are cases when the needed extension is not in this list, or when it's a custom extension developed by the end-user. 
+Adding more extensions is not an easy task in case of a containerized database in Kubernetes-based environment, as normally it would make the user to build a custom PostgreSQL image. 
+
+Still, starting from the Operator version 2.3 there is an alternative way to extend Percona Distribution for PostgreSQL by downloading prepackaged extensions from an external storage on the fly, as defined in the `extensions` section of the Operator Custom Resource.
+
+## Enabling or disabling builtin extensions
+
+Percona Distribution for PostgreSQL [builtin extensions](https://docs.percona.com/postgresql/16/) can be easily enabled or disabled in the `extensions.builtin` subsection of the `deploy/cr.yaml` configuration file as follows:
+
+```yaml
+extensions:
+  ...
+  builtin:
+    pg_stat_monitor: true
+    pg_audit: true
+```
+
+Apply changes after editing with `kubectl apply -f deploy/cr.yaml` command.
+
+!!! note
+
+    For obvious reasons, editing this section and applying it is causing Pods
```

Comment:
@fiowro it's of course up to you but calling it obvious reasons sounds a bit condescending to me. can we just say "Editing this section and applying it will cause Pods to restart"?

Reply:
fixed)